ra/ra.go

```diff
@@ -801,8 +801,15 @@ func (ra *RegistrationAuthorityImpl) checkAuthorizationsCAA(
 		if staleCAA, err := validatedBefore(authz, caaRecheckAfter); err != nil {
 			return err
 		} else if staleCAA {
-			// Ensure that CAA is rechecked for this name
-			recheckAuthzs = append(recheckAuthzs, authz)
+			switch authz.Identifier.Type {
+			case "dns":
+				// Ensure that CAA is rechecked for this name
+				recheckAuthzs = append(recheckAuthzs, authz)
+			case "ip":
+				ra.log.AuditInfof("Skipping CAA recheck for IP address identifier: %s", authz.Identifier.Value)
```

Comment:
Probably don't want to log for every single IP identifier -- that's a lot of logs. If you really want to log something, better to log one line at the end. But I think it's probably fine to log nothing at all here.